service/src/tests/parentchain_block_syncer_test.rs

```diff
@@ -0,0 +1,42 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::{
+	tests::mocks::{
+		parentchain_api_mock::ParentchainApiMock, sidechain_api_mock::SidechainApiMock,
+	},
+	ParentchainBlockSyncer,
+};
+use itp_api_client_extensions::ChainApi;
+use std::sync::Arc;
+
+#[test]
+fn test_number_of_synced_blocks() {
```

Comment:
This is a basic test, which is certainly a good start 👍 For this PR, I think we'll leave it at that, because you basically just moved the existing logic. 
Otherwise I would advise to write more unit tests, that check the various error conditions as well: What happens when the enclave API returns an error? What happens if the parentchain API returns an error? That would require the mocks to be a bit more smart than they are right now.